controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -34,6 +36,19 @@ enum class return_type : std::uint8_t
   ERROR = 1,
 };
 
+enum class configuration_type : std::uint8_t
+{
+  ALL = 0,
+  INDIVIDUAL = 1,
+  NONE = 2,
+};
+
+struct InterfaceConfiguration
+{
+  configuration_type type;
+  std::vector<std::string> names = {};
```

Comment:
names of what? joint names? interface names? combined joint + interface names?

Reply:
It is always interface. There is no "joint" anymore here.
```suggestion
  std::vector<std::string> interface_names = {};
```

Reply:
Doesn't this sounds weird, "interface_configuration.interface_names"? That's kind of redundant in my opinion.
I like `interfae_configuration.names` somewhat more.